components/atoms/Search/search.tsx

```diff
@@ -124,17 +134,22 @@ const Search = ({
       {suggestions && suggestions.length > 0 && showSuggestions && (
         <div className="absolute left-0 z-10 w-full pb-1 space-y-1 bg-white border rounded-lg cursor-pointer shadow-input border-light-slate-6 top-full">
           <ScrollArea type="auto" className="h-60">
+            {suggestionsLabel ? <div className="pl-5 pt-4">{suggestionsLabel}</div> : null}
             {suggestions.map((suggestion, index) => (
               <div
                 className={clsx(
                   cursor === index && "_cursorActive bg-slate-100",
-                  "px-4 py-2 overflow-hidden break-all text-light-slate-9 hover:bg-light-slate-2"
+                  "px-4 py-2 overflow-hidden break-all hover:bg-light-slate-2"
                 )}
                 style={suggestionsStyle}
                 key={index}
-                onClick={() => handleOnSelect(suggestion)}
+                data-suggestion={typeof suggestion === "string" ? suggestion : suggestion.key}
+                onClick={(event) => {
+                  const { suggestion } = (event.target as HTMLElement).dataset;
+                  suggestion && handleOnSelect(suggestion);
+                }}
               >
-                <span className="pl-5 text-sm ">{suggestion}</span>
+                {typeof suggestion === "string" ? <span className="pl-5 text-sm ">{suggestion}</span> : suggestion.node}
```

Comment:
If suggestions are strings, it renders just like it did before. Otherwise, it renders a rich text suggestion.